tests/ci/run_tests

```diff
@@ -76,11 +78,15 @@ main() {
         run_docker_test ./ci/avalon-ganache-proxy-model-wp-tests.yaml
     elif [ "$GANACHE_PROXY_MODEL_WP_SGX_TESTS" == "1" ]; then
         run_docker_test ./ci/avalon-ganache-proxy-model-wp-sgx-tests.yaml
+	elif [ "$MULTIPLE_WP_DIRECT_MODEL_TESTS" == "1" ]; then
+        run_docker_test ./ci/avalon-multiple-wp-direct-model-tests.yaml
+	elif [ "$MULTIPLE_WP_DIRECT_MODEL_SGX_TESTS" == "1" ]; then
+        run_docker_test ./ci/avalon-multiple-wp-direct-model-sgx-tests.yaml
```

Comment:
please align the newly added lines to previous lines. Remove extra spaces added at the beginning.